src/plugins/intel_gpu/src/graph/tile.cpp

```diff
@@ -45,28 +45,26 @@ std::vector<layout> tile_inst::calc_output_layouts(tile_node const& /*node*/, co
     ShapeType repeats_shape = impl_param.input_layouts.size() == 2 ? impl_param.get_input_layout(1).get<ShapeType>()
                                                                    : ov::Shape{ desc->repeats.size() };
     ov::op::v0::Tile op;
-    std::vector<ShapeType> output_shapes = {ShapeType{}};
     std::vector<ShapeType> input_shapes = {
         input0_layout.get<ShapeType>(),
         repeats_shape
     };
 
     auto& constant_mem = impl_param.memory_deps;
+    ov::Tensor repeats_tensor;
     if (constant_mem.count(1)) {
         auto repeats_mem = constant_mem.at(1);
         cldnn::mem_lock<uint8_t, mem_lock_type::read> repeats_lock(repeats_mem, impl_param.prog->get_stream());
-        std::map<size_t, ngraph::HostTensorPtr> const_data = {
-            {1, make_host_tensor(repeats_mem->get_layout(), repeats_lock.data())}
-        };
-        ov::op::v0::shape_infer(&op, input_shapes, output_shapes, const_data);
+        const auto& layout = repeats_mem->get_layout();
+        repeats_tensor =
+            ov::Tensor(data_type_to_element_type(layout.data_type), layout.get_shape(), repeats_lock.data());
```

Comment:
It's illegal change as when `repeats_lock` is deleted, `repeats_lock.data()` ptr used for tensor creation may become invalid.

Reply:
Corrected.